whatToDad_site/whatToDad_app/models.py

```diff
@@ -20,6 +22,22 @@ def __str__(self):
 
 
 class PostLikes(models.Model):
-    user_Id = models.ForeignKey(Author, on_delete= models.CASCADE)
-    post_Id = models.ForeignKey(Post, on_delete= models.CASCADE)
-# Come back to speak about rather we use CASCADE or not
\ No newline at end of file
+    user = models.ForeignKey(Author, on_delete= models.CASCADE)
+    post = models.ForeignKey(Post, on_delete= models.CASCADE)
+# Come back to speak about rather we use CASCADE or not
+
+class Activity(models.Model):
+    detail = models.CharField(max_length=200, unique=True)
+    min_age = models.IntegerField()
+    max_age = models.IntegerField()
+    addressNumber = models.IntegerField()
+    addressStreet = models.CharField(max_length=200)
+    city = models.CharField(max_length=200)
+    state = models.CharField(max_length=200)
+    zip = models.IntegerField()
+    user = models.ForeignKey(Author, on_delete= models.CASCADE)
```

Comment:
Consider using consistent naming for this field. You use `user` here but elsewhere it uses `author` (and previously used `user_Id`)